.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '[Bug]: '
+labels: ['bug', triage']
+assignees: ''
+---
+
+**What steps did you take and what happened:**
+
+[A clear and concise description of what the bug is.]
+
+**What did you expect to happen:**
+
+[Expected outcome listed here.]
+
+**Anything else you would like to add:**
+
+[Miscellaneous information that will assist in solving the issue.]
+
+**Environment:**
```

Comment:
I think architecture could be a helpful thing to add.

Reply:
Also while we only have a limited number of install methods, it might be worth prompting the user to tell us this (e.g., `curl`, brew, nix etc.)

Reply:
I do think that these environment specific asks are a nice extra though. Personally I find it annoying when the issue template asks me to specify 100 different things when I am 99% sure they're not applicable in the specific scenario.